src/python/zquantum/qaoa/problems/_qiskit_problem_helpers.py

```diff
@@ -0,0 +1,68 @@
+import openfermion
+import networkx as nx
+import numpy as np
+from qiskit.optimization.applications.ising import (
+    graph_partition,
+    max_cut,
+    exact_cover,
+    stable_set,
+)
+from zquantum.core.openfermion import qiskitpauli_to_qubitop
+
+
+def _adjacency_matrix(graph: nx.Graph) -> np.ndarray:
+    return nx.to_numpy_array(graph)
+
+
+def _change_matrix_to_qiskit_convention(weights_matrix: np.ndarray):
```

Comment:
Maybe this should just be: `flip_matrix_term_ordering` since you could call it repeatedly no? It would also then make sense to call it to both flip from qiskit -> zquantum and zquantum -> qiskit